src/node/interfaces.cpp

```diff
@@ -170,6 +170,16 @@ class NodeImpl : public Node
         }
         return false;
     }
+#ifdef ENABLE_EXTERNAL_SIGNER
+    std::vector<ExternalSigner> externalSigners() override
```

Comment:
3f845ea2994f53e29abeb3fa158c35f1ee56e7e8

Should we let this throw `runtime_error`? cc @ryanofsky

Reply:
Most of these ifdefs are gone in https://github.com/bitcoin/bitcoin/pull/21935